trust-gh-pages/js/sims/PD.js

```diff
@@ -143,6 +143,10 @@ PD.playOneGame = function(playerA, playerB, agents){
     }
   }
 
+  // AI players are immune to negative punishment
+  if(A.isAI && payoffs[0] < 0) payoffs[0] = 0;
+  if(B.isAI && payoffs[1] < 0) payoffs[1] = 0;
```

Comment:
So this is changing the applied payoff's value to 0? That works, a backup implementation if we need it could be to not apply any payoff if it's negative. Lmk if I'm misunderstanding, as well